data-types/collections/hashes.rb

```diff
@@ -1,3 +1,17 @@
+#hashes.rb contains a series of methods that need to be completed by you.
+
+#For each method,
+#1. Read the comments and attempt to add or modify the code to complete.
+#3. run the file from the terminal to test your code: $ruby hashes.rb
+#4. When it runs, terminal will output some stats:
+ # - # of runs (# of methods run)
+ # - # of assertions (# of assertion tests made)
+ # - # of failures (# of assertions that ran but failed)
+ # - # of errors (# of methods that couldn't run due to an error)
+ # - # of skips (# of methods skipped because the 'skip' command is in the method)
+
+#As you sucessfully complete the methods you'll see your # of assertions increase.  If you see any failures or errors increase it means your method was not modified correctly - revisit your code and try again.
```

Comment:
hey will! Can you try to keep the line length to about 80 characters?

If you have a long line like this:

```ruby
 # As you sucessfully complete the methods you'll see your # of assertions increase.  If you see any failures or errors increase it means your method was not modified correctly - revisit your code and try again.
```

you can make it a multi-line comment:

```ruby
# As you sucessfully complete the methods you'll see your # of assertions increase.
# If you see any failures or errors increase it means your method was not modified 
# correctly, revisit your code and try again.
```

Reply:
will do!